### DISCUSSION TITLE
grand gEAR HRP plan

### DISCUSSION THREAD

**Discussion by RLC-DCPPC** [IGS/gEAR#626] [Ideas]
We need any and all ideas for a 3-5 year plan to make gEAR better and better. 

This will be used to wrote and RO1 style budget and plan that HRP will use to find us the $. Figures like 300-500/yr were bandied about.

How many resources would be plan for? We have about 3.5 (?) FTEs currently.

**Comment by adkinsrs** [comment]
* More plotting options
* Interfacing with other APIs/tools for exporting or importing data
* I would love to be able to migrate from hosting files on a VM to hosting files on a cloud-based file system.  In addition, attempt to move some of the computation tasks off the VM to on-demand scalable series (splitting the monolithic VM into microservices). This would save use money in that we could shrink the VM and call and scale resources as needed.  More research would be needed but it would add some needed stability (and fault-tolerance) to gEAR if implemented right.
  * Another benefit would be that it would be easier to deploy testing after every commit via Github Actions. Or be able to just spawn off the relevant service for testing instead of deploying to the entire (current) development VM

**Comment by adkinsrs** [comment]
From meeting today:

What innovative things can we add to gEAR that can utilize joint collections of datasets?
